docs/astrocut/index.rst

```diff
@@ -133,39 +133,94 @@ TESS FFIs are large and therefore are described by WCS objects that have many no
-or as one fits file per cutout. It is important to remember that while the expectation is
-that all input image are aligned and have the same pixel scale, no checking is done.
+These functions provide general purpose fits cutout functionality, returning the results either
+as fits cutout files, or as images (jpg or png).
+An image normalization (`~astrocut.normalize_img`) function is also available.
+
+The function `~astrocut.fits_cut` takes one or more fits files and performs the same cutout in each,
+returning the result either in a single fits file or as one fits file per cutout.
+It is important to remember that while the expectation is that all input image are
+aligned and have the same pixel scale, no checking is done.
 
 .. code-block:: python
 
                 >>> from astrocut import fits_cut
                 >>> from astropy.io import fits
                 >>> from astropy.coordinates import SkyCoord
-                >>> 
+                
                 >>> input_files = ["https://archive.stsci.edu/pub/hlsp/candels/cosmos/cos-tot/v1.0/hlsp_candels_hst_acs_cos-tot-sect23_f606w_v1.0_drz.fits",
                 ...                "https://archive.stsci.edu/pub/hlsp/candels/cosmos/cos-tot/v1.0/hlsp_candels_hst_acs_cos-tot-sect23_f814w_v1.0_drz.fits"]
 
                 >>> center_coord = SkyCoord("150.0945 2.38681", unit='deg')
                 >>> cutout_size = [200,300]
-                >>> 
-                >>> cutout_file = fits_cut(input_files, center_coord, cutout_size, drop_after="", single_outfile=True)
-                >>> print(cutout_file)
-                cutout_150.094500_2.386810_200x300_astrocut.fits
-
-                >>> cutout_hdulist = fits.open(cutout_file)
-                >>> cutout_hdulist.info()
-                Filename: cutout_150.094500_2.386810_200x300_astrocut.fits
+                
+                >>> cutout_file = fits_cut(input_files, center_coord, cutout_size, drop_after="", single_outfile=True)  
+                >>> print(cutout_file)  
+                ./cutout_150.094500_2.386810_200x300_astrocut.fits
+
+                >>> cutout_hdulist = fits.open(cutout_file)  
+                >>> cutout_hdulist.info()  
+                Filename: ./cutout_150.094500_2.386810_200x300_astrocut.fits
                 No.    Name      Ver    Type      Cards   Dimensions   Format
                   0  PRIMARY       1 PrimaryHDU      11   ()      
                   1  CUTOUT        1 ImageHDU        44   (200, 300)   float32   
                   2  CUTOUT        1 ImageHDU        44   (200, 300)   float32    
 
 
+The function `~astrocut.img_cut` takes one or more fits files and performs the same cutout in each,
+returning the result either an image (jpg or png) per cutout, or a single color image.
+It is important to remember that while the expectation is that all input images are
+aligned and have the same pixel scale, no checking is done.
```

Comment:
Would it be worthwhile to add checks for alignment and pixel scale to your code?

Reply:
Yes, that would definitely be an improvement. It's not straightforward though, so probably warrants its own ticket/issue.